watchman/views.py

```diff
@@ -28,6 +30,49 @@ def status(request):
             response.update(check())
 
     if len(response) == 0:
-        raise Http404('No checks found')
+        raise Http404(_('No checks found'))
 
     return response
+
+
+@token_required
+def dashboard(request):
+    check_types = []
+
+    for check in get_checks(None, None):
+        if callable(check):
+            _check = check()
+
+            for _type in _check:
+                # _check[_type] is a list of dictionaries:
+                # Example: [{'default': {'ok': True}}]
+                statuses = []
+
+                for a in _check[_type]:
+                    status = {}
+
+                    # Systems like storage don't have names
+                    if a == 'ok':
+                        status['name'] = ''
+                        status['status'] = 'OK'
```

Comment:
Can we extract all of these `'OK'` / `'ERROR'` hardcoded strings into `STATUS_OK` and `STATUS_ERROR` constants?